docs/runtime/instantiable-module.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Instantiable Modules"
+---
+
+Substrate runtimes are typically created by composing multiple runtime modules. Many such modules are available for re-use in the Substrate Runtime Module Library, while others may be written with a particular runtime in mind. Many such modules are intended to be re-used in a variety of runtimes. For example most blockchains have notion of a currency and may use the SRML's Balances module. But what if a single runtime wants to use the same module twice? This is exactly what instantiable modules are for.
+
+
+## What is an Instantiable Module?
+An instantiable runtime module is, in most ways, just like any other runtime module. It defines storage items, events, dispatchable calls, and a genesis configuration. But with just a few extra bits of syntax, the module can be made instantiable which means it can be deployed multiple times in the same runtime.
```

Comment:
"But with just a few extra bits of syntax"

Really not a fan of this for a docs page.

Also you did not really explain what an instantiable module is technically.

You should say comment about:
* Multiple module instantiations
* Multiple dispatch routing
* Multiple unique runtime storage

and the other technical aspects of what really an instantiable module is